base/src/Data/Macaw/Analysis/RegisterUse.hs

```diff
@@ -678,6 +679,8 @@ processApp aid app = do
       setAssignVal aid (FrameExpr (o+fromInteger c))
     BVSub _ (FrameExpr o) (IVCValue (BVCValue _ c)) ->
       setAssignVal aid (FrameExpr (o-fromInteger c))
+    BVAnd _ (FrameExpr o) (IVCValue (BVCValue _ c)) ->
+      setAssignVal aid (FrameExpr (o Bits..&. memInt (fromInteger c)))
```

Comment:
For consistency with the code above, you could also write this as:

```suggestion
      setAssignVal aid (FrameExpr (o Bits..&. fromInteger c))
```

Reply:
Agreed!